platform/ui/src/components/studyList/StudyList.js

```diff
@@ -104,10 +104,13 @@ function StudyList(props) {
     smallTableMeta
   );
 
+  if (!ready) {
+    return null;
+  }
```

Comment:
Can we do this in-line by the `return` for this component? Potentially aliased as `translationsAreReady` or something similar?

Reply:
yes :D